packages/wallet/src/wallet.ts

```diff
@@ -223,13 +271,25 @@ export class Wallet<
       message?: ethers.utils.BytesLike
       transactions?: commons.transaction.Transaction[]
       nested?: commons.WalletSignRequestMetadata
+      useEip6492?: true // If true, EIP6492 can be used
```

Comment:
Why do we need these signatures to use EIP-6492? shouldn't this:

```
const childBundle = await this.orchestrator.buildDeployTransaction(metadata)
```

be enough?

Reply:
Right, when sending a tx we should build the deployment tx first. 
This is needed here so that `wallet.signMessage` will apply EIP-6492 when needed. 

Unfortunately this implementation of EIP-6492 only works at the top level. If there are nested undeployed wallets, this won't make a valid signature.